rst.rst

```diff
@@ -103,7 +111,7 @@ Headers
 
 .. parse-headers-off
 
-.. note: The overlines in RST are optional and characters are flexible as long as they are consistent.
+.. note: The overlines and characters are flexible in RST as long as they are consistent.  RST requires strict nesting; even after the hierarchy is established, you can't use, say, a third-tier header right under a first-tier whereas MD allows this.  Also, ``rstcheck`` warns if heading titles are not unique (`see this issue <https://github.com/myint/rstcheck/issues/76>`__)
```

Comment:
I'm trying to understand why this rstcheck mention should be here?  This sheet isn't going to be able to describe every possible hiccup.

Reply:
It's something that has bitten me; I could drop it if it's too detailed for this cheat-sheet